### ISSUE TITLE
Update Data Types and Tags based Timefilter selected

### ISSUE THREAD

**Issue by hkhalifa** [google/timesketch#2059]
It would be very helpful to have the "Data Types" and "Tags" updated based on the data present in the range of the active Timefilter

**Describe the solution you'd like**
After setting a Timefilter and then clicking on the search bar, you would have a list of "Data Types" and "Tags" that are present within this time range. Also the numbers beside each "Data Type" and "Tag" should be updated accordingly.

**Comment by berggren** [comment]
Yes, I agree. This would be much more expected from a UX point of view. Let's see if we can get this in in the December release.
